app/media/src/tests/test_server.py

```diff
@@ -68,6 +68,7 @@ def client_with_secrets(tmp_path):
         main_server_address="localhost:8088",
         main_server_use_ssl=False,
         media_upload_location=tmp_path,
+        media_cors_origin="*",
```

Comment:
This is allowing upload from anywhere right? I wonder if it should be more specific to our media server?

We could do something like 

```
const allowedOrigins = ['https://couchers.org', 'http://localhost:3000']; // etc
```

Or am I misunderstanding?

Reply:
Yes, that's right. When running this properly, it'll pull the origin from an environment variable. We'll use something quite restrictive in prod, but in staging it can be `*`.